engines/osxnative/config-schema.yml

```diff
@@ -0,0 +1,37 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              Config
+description: |-
+  Config applicable to Mac OSX native engine
+type:               object
+properties:
+  workerGroup:
+    title:          Worker Group
```

Comment:
Another reason why this is in the worker level config is because we will have a configuration  provider reading data from things like aws and such to determine these values, and these values will be used not only with the engine but elsewhere.